acme/resource_acme_certificate_test.go

```diff
@@ -37,7 +37,7 @@ func TestAccACMECertificate_basic(t *testing.T) {
 		ProviderFactories: testAccProviders,
 		ExternalProviders: testAccExternalProviders,
 		CheckDestroy:      testAccCheckACMECertificateStatus("acme_certificate.certificate", certificateStatusRevoked),
-		Steps: []resource.TestStep{
+			Steps: []resource.TestStep{
```

Comment:
@kingcdavid were these test files run though [`gofmt`](https://pkg.go.dev/cmd/gofmt)? This should likely be set up by your editor, but this indent looks out of place.